### PR TITLE
Add download and permission change instructions to setup script

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/utils/sui-setup.sh [hunk 1]
```diff
@@ -3,7 +3,12 @@
 #!/bin/bash
 # Prereqs: Rust Cargo, Git CLI, and GitHub account
 # Usage: set up environment for Sui development
-# Run `sui-setup.sh` in the directory to download source
+# First run these commands tow download the script
+# curl https://raw.githubusercontent.com/MystenLabs/sui/main/doc/utils/sui-setup.sh -o sui-setup.sh
+# chmod 755 sui-setup.sh
```
Comment (huitseeker) [inline doc/utils/sui-setup.sh:8]: If someone is reading these lines, shouldn't they have gone through this already?

Comment (Clay-Mysten) [reply doc/utils/sui-setup.sh:8]: I am hoping to link to this file and yes reiterate these steps pre-download. But until that happens, we need this documented somewhere.

I haven't gotten any sort of formal OK to recommend this from our docs... WDYT?

**Review by huitseeker — COMMENTED** [review]
I think those instructions make a ton more sense in #1981. Let's discuss there and close this one.

**Comment by Clay-Mysten** [comment]
> I think those instructions make a ton more sense in #1981. Let's discuss there and close this one.

Ack. Closing.

**Pull request closed without merging**